openlibrary/admin/numbers.py

```diff
@@ -132,7 +131,7 @@ def admin_range__visitors(**kargs):
         sqlitefile = tempfile.mktemp(prefix="sqlite-")
         url = "http://www.archive.org/download/stats/numUniqueIPsOL.sqlite"
         logging.debug("  Downloading '%s'", url)
-        sqlite_contents = urllib.request.urlopen(url).read()
+        sqlite_contents = requests.get(url).content
         f = open(sqlitefile, "w")
         f.write(sqlite_contents)
         f.close()
```

Comment:
```suggestion
        with open(sqlitefile, "w") as f:  # Or "wb" is you decide that is best
            f.write(sqlite_contents)
```
This context manager approach is always safer because it will automatically close the file handle even if an exception is thrown in the read operation.

Reply:
@cclauss for `update_work` and `test_update_work`, #3696 3696